pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -115,6 +115,19 @@ rec {
         ]
       ];
 
+      defaultHardeningFlags = [
```

Comment:
See also `supportedHardeningFlags` above. Just so you are aware.

Reply:
Ah, I could just subtract `{ "pie" }` from that set, if that's what you're hinting at.